KNSB/apirest/src/main/java/com/restapi/apirest/rest/Customer.java

```diff
@@ -0,0 +1,21 @@
+package com.restapi.apirest.rest;
+
+
+public class Customer {
+    private String action;
```

Comment:
for a Customer class these 3 properties don't make sense. Action, surname and line?